submissions/Vitamin/port-sniffer/sniffer.js

```diff
@@ -0,0 +1,100 @@
+const Net = require('net');
+const dns = require('dns').promises;
+const { help, isDefined } = require('./helpers');
+
+const [, , ...processArgs] = process.argv;
+const [minPortValue, maxPortValue] = [0, 65535];
+
+const ipToURL = async host => {
```

Comment:
Seems like this function does the opposite to it's name: it resolves IP from hostname.